frame/dex-router/src/lib.rs

```diff
@@ -323,4 +324,15 @@ pub mod pallet {
 			Ok(dx_t)
 		}
 	}
+
+	impl<T: Config> Contains<(CurrencyPair<T::AssetId>, T::PoolId)> for Pallet<T> {
+		fn contains(&(asset_pair, pool_id): &(CurrencyPair<T::AssetId>, T::PoolId)) -> bool {
```

Comment:
So this will only used to check EnabledPoolTWAP? and that route will always contain a single uniswap pool right? if this is correct then better to add some comments here.